webserver.go

```diff
@@ -0,0 +1,66 @@
+package links
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+func webServerCheck(w http.ResponseWriter, r *http.Request) {
+	queryString := r.URL.Query()
+	qsSite := queryString["site"]
+	if len(qsSite) == 0 {
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintln(w, `{"err":"cannot find site in query string"`)
+		return
+	}
+	var noRecursion bool
+	var err error
+	qsNoRecursion := queryString["no-recursion"]
+	if len(qsNoRecursion) > 0 {
+		noRecursion, err = strconv.ParseBool(qsNoRecursion[0])
+		if err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+			fmt.Fprintln(w, `{"err":"cannot convert no-recursion to boolean"`)
+			return
+		}
+	}
+	output := io.Discard
+	qsOutput := queryString["output"]
+	if len(qsOutput) > 0 {
+		output = os.Stdout
```

Comment:
Then what's `qsOutput` for?

Reply:
yeah... it's odd... but I need to be able to set output to stdout for troubleshooting.